src/app/CommandSender.h

```diff
@@ -375,6 +376,22 @@ class CommandSender final : public Messaging::ExchangeDelegate
 
     TLV::TLVWriter * GetCommandDataIBTLVWriter();
 
+    /**
+     * API for adding request data using DataModel::EncodableToTLV.
+     *
+     * @param [in] aCommandPath  The path of the command being requested.
+     * @param [in] aEncodable - an encodable that places the command data structure
+     *             for `aAddRequestDataParams` into a TLV Writer.
```

Comment:
That's not what it does, though....  aEncodable never even sees aAddRequestDataParams.

Reply:
Fixed. I am sorry, about this I should have been better here.